units/commit_queue.go

```diff
@@ -110,7 +110,20 @@ func (j *commitQueueJob) Run(ctx context.Context) {
 		return
 	}
 	nextItem := cq.Next()
-	if nextItem == nil || cq.Processing {
+	if nextItem == nil {
+		return
+	}
+	if cq.Processing {
+		if time.Since(cq.ProcessingUpdatedTime) > time.Hour {
+			grip.Info(message.Fields{
```

Comment:
should it be at a higher level?

Reply:
Is there a way to make it only log the message once when we first detect the queue is stuck?

Reply:
grip.InfoWhen() perhaps, 

I would either: 

make this a grip.Critical so it goes to slack *or* log it always, and just log time.Since(cq.ProcessingUpdatedTime).Seconds()

Reply:
Given that there isn't an easy way to only log once, I opted for keeping the logging level at Info, but always logging.